UIKItTemplate/Modules/ProfileDetails/View/InfoTextField.swift

```diff
@@ -0,0 +1,60 @@
+// InfoTextField.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Шаблон переиспользуемой ячейки ( textField ) для ввода личных данных
+final class InfoTextField: UITextField {
+    var placeholderText: String?
+    var typeOfKeyboard: UIKeyboardType?
+
+    // MARK: - Initializers
+
+    init(placeholderText: String?, typeOfKeyboard: UIKeyboardType) {
+        super.init(frame: .zero)
+        self.placeholderText = placeholderText
+        self.typeOfKeyboard = typeOfKeyboard
+        commonInit()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        commonInit()
+    }
+
+    // MARK: - Private Methods
+
+    private func commonInit() {
+        setupTextField()
+        setupPadding()
+        configureTextField()
+    }
+
+    private func setupTextField() {
+        placeholder = placeholderText
+        backgroundColor = .textFieldBackground
+        textColor = UIColor.textFieldForeground
+        font = .systemFont(ofSize: 16, weight: .bold)
+        layer.cornerRadius = 12
+        guard let typeOfKeyboard = typeOfKeyboard else { return }
```

Comment:
```suggestion
        guard let typeOfKeyboard else { return }
```
Если Xcode не старый - должно работать